docs/articles/expensify-classic/account-settings/Notification-Troubbleshooting.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Notification Troubleshooting
+description: some description here
```

Comment:
@alexpensify we should update the description here

Reply:
I updated!